values.yaml

```diff
@@ -203,10 +223,20 @@ server:
   # By default no direct resource request is made.
 
   image:
-    repository: "vault"
-    tag: "1.7.2"
+    repository: ""
+    tag: ""
     # Overrides the default Image Pull Policy
     pullPolicy: IfNotPresent
+    # Kubernetes defaults (these will be used if global.openshift=false and
+    # OpenShift is not detected)
+    kubernetes:
```

Comment:
Would it be possible to keep the non-openshift config keys in the same place as before? Having `kubernetes` here feels a little odd to me, and I think it could cause confusion for those not familiar with openshift and and not sure whether it's relevant to them.

I assume you've done it this way so if a user specifies `server.image.repository` but deploys to openshift, their value doesn't get ignored? But IMHO I think that would be ok - it seems reasonable that in the default case we always use `server.image.repository`, unless you're deploying to openshift in which case we always use `server.image.openshift.repository`.

Reply:
Yeah I structured it like this because there doesn't seem to be a way to tell if a user has overridden the default for a value. So this way preserves the existing behavior of using `server.image.repository` if a user has it in their chart deployment overrides. I was err'ing on the side of caution since folks are already using the chart on openshift.

Reply:
Good point, I hadn't thought about that upgrade flow 🤔